src/components/organisms/About/About.jsx

```diff
@@ -1,15 +1,12 @@
 import AboutImg from "../../molecules/AboutImg/AboutImg";
+import AboutText from "../../molecules/AboutText/AboutText"
 import "./About.scss";
 
 const About = (props) => {
   return ( 
     <div className="about"> 
-      <AboutImg image = {props.data.about.aboutImages}/>
-      <div className="about-text-container">
-        <h1 className="heading">{props.data.about.title}</h1>
-        <h2 className="subheading">{props.data.about.subTitle}</h2>
-        <p className="about__paragraph">{props.data.about.text}</p>
-      </div>
+      <AboutImg image = {props.data.about.aboutImages}/>     
```

Comment:
Same. Why are you making a separate `AboutImg` component for the text?

Reply:
I needed to use atomic design patterns, as I read about it it can be done like that as atoms is 1 image and 2 images are molecule. If it needs to be changed , I will do that

Reply:
I do not see the point in the `AboutText` and `AboutImg` components, since the `About` component itself is empty and therefore it does not need to be simplified by creating additional components.

Reply:
Changed